config/default.yml

```diff
@@ -349,3 +349,12 @@ Sorbet/ForbidComparableTEnum:
   Description: 'Disallows including the `Comparable` module in a `T::Enum`.'
   Enabled: true
   VersionAdded: 0.8.2
+
+Sorbet/SelectByIsA:
+  Description: >-
+    Suggests using `grep` over `select` when using it only for type narrowing.
+    This is because Sorbet can properly infer types when using `grep` but not with `select`.
+  Enabled: pending
```

Comment:
Should we just go with `true` straight away?

Reply:
We can, but that isn't how historically rubocop plugins works. They consider enabling cops breaking changes so they require major version bumps. That is why they introduce cops as pending in minor versions. But we are pre 1.0, so I have no problem on making breaking changes